www/js/menu.js

```diff
@@ -301,6 +304,68 @@ const announcementSlideButton = h(
   ),
 );
 
+const createSyncModal = (content, code) => {
+  const syncModal = new tingle.Modal({
+    footer: true,
+    stickyFooter: false,
+    cssClass: ['sync-code-modal'],
+    closeMethods: ['overlay', 'button', 'escape'],
+  });
+  // open modal
+  syncModal.open();
+
+  if (code) {
+    syncModal.addFooterBtn('Copy', 'tingle-btn tingle-btn--primary', () => {
+      copy(code);
+      syncModal.close();
+      syncModal.destroy();
+    });
+    syncModal.addFooterBtn('End Session', 'tingle-btn tingle-btn--primary', () => {
+      onEnd(code);
+      syncModal.close();
+      syncModal.destroy();
+      document.getElementById('remote-sync-icon').style.color = '#424242';
+    });
+    global.controller.sendText(code);
+  }
+
+  // set content
+  syncModal.setContent(content);
+  // add cancel button
+  const cancelTitle = 'Cancel';
+  syncModal.addFooterBtn(
+    cancelTitle,
+    'tingle-btn tingle-btn--pull-right tingle-btn--default',
+    () => {
+      syncModal.close();
+      syncModal.destroy();
+    },
+  );
+};
+
+const remoteSyncButton = h(
+  'li',
+  h(
+    'a.remote-sync-button',
+    {
+      onclick: async () => {
+        const onlineVal = await isOnline();
+        if (onlineVal) {
+          const code = await tryConnection();
+          if (code) {
+            document.getElementById('remote-sync-icon').style.color = 'green';
```

Comment:
`nitpicking` We don't use html green anywhere else, maybe use the green we use otherwise?